x/marker/spec/12_transfers.md

```
@@ -211,6 +211,8 @@ flowchart TD
     linkStyle 5,7,8 stroke:#1b8500,color:#1b8500
 ```
 
+The `require_deposit_access` opt-in is only consulted in the [SendRestrictionFn](#SendRestrictionFn) (i.e. bank send) path. In the [MsgTransferRequest](#MsgTransferRequest) path, the deposit-access check is triggered only when the receiver is a restricted marker, regardless of its `require_deposit_access` value.
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)